### ISSUE TITLE
Increase max upload size, or sync it with virtualbox.memory

### ISSUE THREAD

**Issue by johnbillion** [Chassis/Chassis#640]
The intention of the `post_max_size` and `upload_max_filesize` directives in PHP is to prevent giant files from being uploaded and eating up all the server's resources. For a local development environment, it's safe to jack up this limit as long as the VM can handle it.

Chassis currently sets a limit of 100MB for `upload_max_filesize`, `post_max_size`, and the corresponding `client_max_body_size` directive for Nginx. The `memory_limit` directive appears to be set to 128M, although my local sites have it set to 1024M and I can't remember if I changed that manually.

I think these default limits should be increased to 1GB all round. We've found that the kind of sites that we're building are using infrastructure (and processes) that support much higher limits, and upping these limits in Chassis for testing is quite a pain, especially for inexperienced users.

Alternatively, these limits could be dynamically populated based on the value of `virtualbox.memory` in config.yml.

The `Chassis/phpini` extension exists, but this doesn't fix the `client_max_body_size` directive for Nginx.

Thoughts?

**Comment by paulgibbs** [comment]
+1 this. I edit these way higher on most of the projects I work on anyway.

**Comment by BronsonQuick** [comment]
@johnbillion Good suggestion Sir and thanks for the +1 @paulgibbs!

I've whipped up a PR for this. I've added `upload_size: 1024M` to the the `yaml` config. This sets``post_max_size`` and ``upload_max_filesize`` in ``php.ini`` and sets ``client_max_body_size`` in Nginx.

I didn't want to use the VirtualBox memory setting because a) that's more to do with memory and b) some people use VMWare. I'll tag Ryan in the review to see what his thoughts are.
